canvas_modules/common-canvas/src/common-properties/controls/expression/expression.scss

```diff
@@ -35,6 +35,10 @@
 	border-left: 1px solid $text-01;
 }
 
+.CodeMirror-gutters {
+	border-right: none;
```

Comment:
We should also add `background-color: inherit;`.  This way the background of the numbers match the panel.